users/README.md

```diff
@@ -303,3 +304,28 @@ Updates roles for the User.
   - **Code:** 500
     - **Content:**
       `{ 'statusCode': 500, 'error': 'Internal Server Error', 'message': 'An internal server error occurred' }`
+
+## **PATCH /users/update-archived**
+
+Archive users if not in Discord.
+
+- **Params**  
+  None
+- **Query**  
+  None
+- **Headers**  
+  Content-Type: application/json
+- **Cookie**  
+  rds-session: `<JWT>`
+- **Body**  
+  None
+- **Success Response:**
+  - **Code:** 200
+    - **Content:** `{ 'message': 'Successfully updated users archived role to true if in_discord role is false'}`
+- **Error Response:**
+  - **Code:** 401
+    - **Content:**
+      `{ 'statusCode': 401, 'error': 'Unauthorized', 'message': 'Unauthenticated User' }`
+  - **Code:** 500
+    - **Content:**
+      `{ 'statusCode': 500, 'error': 'Internal Server Error', 'message': 'An internal server error occurred' }`
```

Comment:
please use multi line statements in code blocks mentioning proper language
it gives a cleaner look to the document.

Also add some examples.